lib/LaTeXML/resources/CSS/LaTeXML.css

```diff
@@ -443,6 +443,16 @@ span.ltx_framed       { display:inline-block; text-indent:0; } /* avoid padding/
 .ltx_svg_fog foreignObject > p { margin:0; padding:0; display:block; }
 /*.ltx_svg_fog foreignObject > p { margin:0; padding:0; display:block; white-space:nowrap; }*/
 
+.ltx_foreignobject_container { display:flex; width:100%; height:100%; align-items: end; justify-content: center; justify-items:center; }
+@supports not (text-box: trim-both cap alphabetic) {
+  .ltx_foreignobject_content { line-height: calc(var(--lx-foreignobject-height) - var(--lx-foreignobject-depth)); translate: 0 var(--lx-foreignobject-depth); }
+  .ltx_foreignobject_content .ltx_inline-block { line-height: initial; } }
+@supports (text-box: trim-both cap alphabetic) {
+  .ltx_foreignobject_content { text-box: trim-both cap alphabetic; } }
+/* workaround for Firefox setting line-height:normal on math elements */
+@supports (-moz-appearance: none) and (not (text-box: trim-both cap alphabetic)) {
+  .ltx_foreignobject_content math { line-height: inherit; } }
```

Comment:
It turns out that the Firefox default CSS has a `math { line-height: normal; }` rule which negates the effect of the previous `line-height` rule. Unfortunately, the content is still too high in Firefox.